### PR TITLE
6.0.10 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_6.0.md [hunk 1]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+	- Lifetime used
+	- Power On Hours
+	- TBW (Terabyte Written)
+	- TBW per year
+------
+- **I95-49824 BFD Overlay over SVR** In deployments where the number of SVR sessions between SSRs are limited due to carrier settings, the established BFD channel is leveraged to encapsulate SVR sessions. For details about using this feature, see [BFD Overlay over SVR.](config_bfd_tunnel.md)
+------
+- **I95-50072 Support for ConnectX-6 Lx PCIe device:** Support has been added for this device.
+------
+- **I95-50510 New fields for IPFIX:** The SSR IPFIX implementation was not sending the industry standard fields of flowStartMilliseconds and flowEndMilliseconds. In the new implementation, all IPFIX records include these fields. The start time is set to the start time of the flow, and the end time is always set to the time the last packet was received on the flow. For intermediate record, this indicates that the flow is still ongoing but provides the last activity timestamp. For the end record, this indicates when the last packet was received on the flow prior to the session terminating. For additional information, see [IPFIX](concepts_application_discovery.md#ipfix).
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:69]: "For Intermediate records, ..."
need an 's' on record
Also, this needs to be the same as 6.1.2 release notes.

File: docs/release_notes_128t_6.0.md [hunk 2]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
+- **I95-50571 Add packet buffer tracking to help analyze buffer exhaustion:** The following features have been added to help diagnose frequent packet buffer pool depletions in customer environments:
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:71]: Remove the work "frequent" and change "customer" to "certain".

File: docs/release_notes_128t_6.0.md [hunk 3]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+Please refer to [Password Policies](config_password_policies.md) for updated password requirements.
+:::
+------
+- **I95-42379 BGP over SVR global service policy:** [Security and Service Policy](config_bgp.md#security-policy-and-service-policy) configuration options are provided for specifying the policy to be used for generated BGP-over-SVR services.
+------
+- **I95-48485 Broadcom NIC (NetXtreme) fails to initialize properly:** Resolved an issue with initization errors during memzone creation.
+------
+- **I95-48518 Application Identification not recognizing Apps:** Resolved an issue where the GUI was only pulling Application data from one node in an HA configuration. Application ID Summary display now aggregates data from both nodes.
+------
+- **I95-48862 Load balance sessions across BGP RIB Entries with multiple paths:** Resolved an issue when BGP was used to build a routing table, only the first next hop was used. All next hops are now used, and load balancing occurs over all routing protocol routes.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:94]: In 6.1.2 release notes, I believe we moved this to the feature section.

File: docs/release_notes_128t_6.0.md [hunk 4]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
+ - **I95-49340 Crash when the unexpected input of `tenant-prefix` with no `source-address` is committed:** Validation has been added to restrict the `tenant-prefix source-address` to a minimum of one.
+------
+-  **I95-49350 BFD echo generating latency overhead:** BFD echo tests are now staggered to minimize application latency's contribution to overall peer path latency.
+------
+- **I95-49377 Sessions established, packet counters incremented, seen in device-capture, but not seen by next-hop:** Added hooks for NIC driver to trigger an unrecoverable event and invoke the Highway lockup detector mechanism.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:100]: Not sure why I didn't comment on this header for the 5.6.7 release notes. The heading needs to be changed for both.
**I95-49377 Transmit packets dropped by NIC for established sessions - packet counters are incrementing and can be seen in packet capture, but not seen by next-hop:**

File: docs/release_notes_128t_6.0.md [hunk 5]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
+- **I95-49675 Incorrect path in console help message for `export config running`:** The help message now correctly identifies the export path: `Exported files are stored in /etc/128technology/config-exports/ and are stored as GZIP compressed files.`
+------
+- **I95-49754 Waypoint re-use causing duplicate reverse flows:** Resolved a case where when the waypoint pool is nearly depleted, released waypoints were reused prematurely causing errors when installing reverse flows.
+------
+- **I95-49791 Add audit rules to track modification of grub config files:** Added rules to log notifications in case of changes to grub configuration files.
+------
+- **I95-49925 GRE tunnel health-check not migrating sessions when path is down:** The GRE tunnel manager now removes all sessions before adding new ones rather than modifying the existing sessions.
+------
+- **I95-49969 Permission Denied error when attempting to self-generate a webserver certificate:** Resolved an issue that prevented users with the admin role from creating a new self-signed web certificate via the PCLI command create certificate self-signed webserver.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:122]: Please cross reference changes made to 6.1.2 release notes. Comment for those release notes were to add back ticks around `create certificate self-signed webserver`

File: docs/release_notes_128t_6.0.md [hunk 6]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
+- **I95-50050 VRRP High Availability gets stuck in Active/Active:** The DPDK version has been updated to resolve this issue.
+------
+- **I95-50247 Duplicate peer path alarms:** Resolved an issue where both BFD and the path MTU feature were generating alarms for the same peer path being down. The criteria for which peerPath state changes can trigger peer path events has been tightened.
+------
+- **I95-50260 `show idp events` does not honor the `router` or `node` arguments:** Resolved an issue where `show idp events` did not honor the `router` and `node` arguments and always executed against the local node. The command is now executed correctly, using the specified arguments.
+------
+- **I95-50262 Unconnected routers not rotating logs often enough:** Resolved an issue where a managed router was not able to pull down the configuration from the Conductor - which includes the log rotation config. The default salt log rotation configuration has been improved, preventing the log from growing too large before the connection to the Conductor can be established.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:138]: **I95-50262 Routers disconnected from their conductor may have incorrect log rotation settings:**

File: docs/release_notes_128t_6.0.md [hunk 7]
```diff
@@ -50,6 +50,146 @@ The Juniper SSR team does not publicly disclose known or resolved CVEs in our pu
...
+------
+- **I95-51007 Conductor is incorrectly honoring core pinning:** The `cpuProperties` cores setting in `/etc/128technology/local.init` was erroneously isolating cores on conductor nodes when set, even though this setting is intended for a router. This would cause a reduction in available processing cores for normal conductor operations. This setting will now be ignored on the conductor.
+------
+- **I95-51021 Package to Image conversion fails on FIPS enabled SSR:** Conversion of package-based to image-based is now supported for systems with FIPS 140-2 mode enabled.
+------
+- **I95-51044 Hide forwarding-core-mode on conductor:** Disabled the f`orwarding-core-mode` setting on conductor nodes, since this setting doesn't apply to conductor.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:186]: 'f' needs to be contained within `forwarding-core-mode`

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md (hunk 4 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md]: changed in both.

File: docs/release_notes_128t_6.0.md (hunk 5 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md]: missed this one, changed in 6.1.2

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**